draft-ietf-webtrans-http2.md

```diff
@@ -924,6 +932,113 @@ Specification:
 
 : This document
 
+The SETTINGS_WEBTRANSPORT_INITIAL_MAX_DATA parameter indicates the initial value
+for the session data limit, otherwise communicated by the WT_MAX_DATA capsule
+(see {{WT_MAX_DATA}}). The default value for the
+SETTINGS_WEBTRANSPORT_INITIAL_MAX_DATA parameter is "0", indicating that the
+endpoint will send a WT_MAX_DATA capsule within each session before its peer is
+allowed to send any stream data within that session.
+
+Note that this limit applies to all WebTransport sessions that use the HTTP/2
+connection on which this SETTING is sent.
+
+Setting Name:
+
+: SETTINGS_WEBTRANSPORT_INITIAL_MAX_DATA
+
+Value:
+
+: 0x2b603744
+
+Default:
+
+: 0
+
+Specification:
+
+: This document
+
+The SETTINGS_WEBTRANSPORT_INITIAL_MAX_STREAM_DATA parameter indicates the
+initial value for the stream data limits, otherwise communicated by the
```

Comment:
Don't we want to split this?  WT_MAX_STREAM_DATA applies to each stream individually, but you might want to have different bidirectional and unidirectional stream handling.

Reply:
Good call, split these out in [5dec3cb](https://github.com/ietf-wg-webtrans/draft-ietf-webtrans-http2/pull/78/commits/5dec3cbe55823ef3f384a3e65ad2f6f985c0e3f6)